.github/workflows/weblate.yml

```diff
@@ -0,0 +1,20 @@
+name: Weblate
+
+on:
+  push:
+    branches:
+      - master
+      - weblate
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - run: sudo apt-get install intltool curl
+      - run: intltool-update --pot
```

Comment:
I just dropped intltool so I'll merge this one first and then update the workflow.